core/src/test/resources/test_tumor_type_summaries.txt

```diff
@@ -65,32 +65,32 @@ BRAF	MKRN1-BRAF Fusion	Breast Cancer	BRAF, an intracellular kinase, is frequentl
+BRAF	V600D	Non-Small Cell Lung Cancer	BRAF, an intracellular kinase, is frequently mutated in melanoma, thyroid and lung cancers among others.	The BRAF V600D mutation is known to be oncogenic.	While the RAF-targeted inhibitor dabrafenib in combination with the MEK1/2-targeted inhibitor trametinib is FDA-approved for the treatment of patients with BRAF V600E mutant melanoma, non-small cell lung cancer and anaplastic thyroid cancer, the clinical utility of dabrafenib in combination with trametinib in patients with BRAF V600D mutant non-small cell lung cancer has yet to be defined.
+BRAF	V600G	Non-Small Cell Lung Cancer	BRAF, an intracellular kinase, is frequently mutated in melanoma, thyroid and lung cancers among others.	The BRAF V600G mutation is likely oncogenic.	While the RAF-targeted inhibitor dabrafenib in combination with the MEK1/2-targeted inhibitor trametinib is FDA-approved for the treatment of patients with BRAF V600E mutant melanoma, non-small cell lung cancer and anaplastic thyroid cancer, the clinical utility of dabrafenib in combination with trametinib in patients with BRAF V600G mutant non-small cell lung cancer has yet to be defined.
 BRAF	V600E	Gastrointestinal Stromal Tumor	BRAF, an intracellular kinase, is frequently mutated in melanoma, thyroid and lung cancers among others.	The BRAF V600E mutation is known to be oncogenic.	While the RAF-targeted inhibitor dabrafenib in combination with the MEK1/2-targeted inhibitor trametinib is FDA-approved for the treatment of patients with BRAF V600E mutant melanoma, non-small cell lung cancer and anaplastic thyroid cancer, the clinical utility of dabrafenib in combination with trametinib in patients with BRAF V600E mutant gastrointestinal stromal tumors has yet to be defined.
-BRAF	V600K	Gastrointestinal Stromal Tumor	BRAF, an intracellular kinase, is frequently mutated in melanoma, thyroid and lung cancers among others.	The BRAF V600K mutation is known to be oncogenic.	While the MEK1/2-targeted inhibitor trametinib, alone or in combination with the RAF-targeted inhibitor dabrafenib, and the MEK-targeted inhibitor cobimetinib in combination with the RAF-targeted inhibitor vemurafenib are FDA-approved for the treatment of patients with BRAF V600K mutant melanoma, their clinical utility in patients with BRAF V600K mutant gastrointestinal stromal tumors has yet to be defined.
+BRAF	V600K	Gastrointestinal Stromal Tumor	BRAF, an intracellular kinase, is frequently mutated in melanoma, thyroid and lung cancers among others.	The BRAF V600K mutation is known to be oncogenic.	While the RAF-targeted inhibitor dabrafenib in combination with the MEK1/2-targeted inhibitor trametinib is FDA-approved for the treatment of patients with BRAF V600E mutant melanoma, non-small cell lung cancer and anaplastic thyroid cancer, the clinical utility of dabrafenib in combination with trametinib in patients with BRAF V600K mutant gastrointestinal stromal tumors has yet to be defined.
```

Comment:
I think this should get the summary under V600K other tumor types - instead, it is getting the summary under V600 other tumor types